hadoop-yarn-project/hadoop-yarn/hadoop-yarn-api/src/main/java/org/apache/hadoop/yarn/conf/YarnConfiguration.java

```diff
@@ -4001,7 +4001,7 @@ public static boolean isAclEnabled(Configuration conf) {
       FEDERATION_PREFIX + "state-store.class";
 
   public static final String DEFAULT_FEDERATION_STATESTORE_CLIENT_CLASS =
-      "org.apache.hadoop.yarn.server.federation.store.impl.MemoryFederationStateStore";
+      "org.apache.hadoop.yarn.server.federation.store.impl.ZookeeperFederationStateStore";
```

Comment:
Do we have this also in some of the XMLs?
The other thing is we could use the getClass() and getName()

Reply:
Thank you for helping to review the code! I will check the configuration in xml.